iP/src/duke/Duke.java

```diff
@@ -1,6 +1,7 @@
 package duke;
```

Comment:
Maybe you can have a new empty line indentation after your package line before your import statements?

Reply:
Repeated suggestion. I second this as well.